web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -122,6 +123,15 @@ public GraphHopperManaged(GraphHopperConfig configuration, ObjectMapper objectMa
         }
         configuration.setProfiles(newProfiles);
 
+        // Error if gtfs_link_mapper profile wasn't included in GH config (link mapper step will fail in this case)
+        if (configuration.has("gtfs.file")) {
+            if (newProfiles.stream()
+                    .filter(p -> p.getName().equals("gtfs_link_mapper"))
+                    .collect(Collectors.toSet()).size() == 0) {
```

Comment:
nit: you can just use `.count()` on the stream rather than collecting it and checking the size of the set

Reply:
Ahh good call! Will update 👍